webview/cocoa.py

```diff
@@ -244,15 +290,25 @@ def __init__(self, title, url, width, height, resizable, fullscreen, min_size, b
         if resizable:
             window_mask = window_mask | AppKit.NSResizableWindowMask
 
+        if frameless:
+            window_mask = window_mask | AppKit.NSFullSizeContentViewWindowMask | AppKit.NSTexturedBackgroundWindowMask
```

Comment:
`NSFullSizeContentViewWindowMask` is now [deprecated](https://developer.apple.com/documentation/appkit/nsfullsizecontentviewwindowmask). Doesn't work in High Sierra. If
we are to use this, we would need an OS API version check. Also a way to do it in 10.13+